srcpkgs/linux4.19/files/ppc64le-dotconfig

```diff
@@ -618,9 +636,9 @@ CONFIG_REFCOUNT_FULL=y
 # GCOV-based kernel profiling
 #
 # CONFIG_GCOV_KERNEL is not set
-# CONFIG_ARCH_HAS_GCOV_PROFILE_ALL=y
-# CONFIG_PLUGIN_HOSTCC is not set
-# CONFIG_HAVE_GCC_PLUGINS is not set
+CONFIG_ARCH_HAS_GCOV_PROFILE_ALL=y
+CONFIG_PLUGIN_HOSTCC="g++"
+CONFIG_HAVE_GCC_PLUGINS=y
```

Comment:
These 3 lines above keep coming back while they were (and probably should be) disabled.

Reply:
these are added in by the xbps-src build environment, i specifically made sure to reconfigure the kernels within xbps-src so that it does not ask for anything

Reply:
xbps-src build environment does not ask me when I have `# CONFIG_HAVE_GCC_PLUGINS is not set`, though?

Reply:
How do I disable that, though? It's not on menuconfig, it gets picked automatically

Reply:
I think it may be the `make … oldconfig` which brings them back into `.config`.

Reply:
Heh, my `/boot/config-4.19.36_1` has:
```
CONFIG_ARCH_HAS_GCOV_PROFILE_ALL=y
CONFIG_PLUGIN_HOSTCC="g++"
CONFIG_HAVE_GCC_PLUGINS=y
# CONFIG_GCC_PLUGINS is not set
```
so plugins are enabled anyway.

Reply:
Yeah, as long as `CONFIG_GCC_PLUGINS` is not set, it should not matter what the rest of them are set to.

Reply:
yes. as long as `CONFIG_GCC_PLUGINS` is off, it should not matter